### ISSUE TITLE
Switch to Prometheus Operator

### ISSUE THREAD

**Issue by falfaro** [bitnami/kube-prod-runtime#529]
Switch to using Prometheus Operator and use CRDs for rules and alerting instead.

**Comment by irizzant** [comment]
I totally agree with this, IMHO Prometheus Operator also offers better Grafana dashboards for cluster monitoring.
